src/parser/core/modules/Medicated.ts

```diff
@@ -43,14 +43,20 @@ export class Medicated extends Analyser {
 			throw new Error('potion instance not found')
 		}
 
-		this.pots.push({end: event.timestamp, ...this.pot})
+		this.pots.push({ ...this.pot, end: event.timestamp })
```

Comment:
Why?

Reply:
Ahh was just a readability thing based on personal preference, but when spreading values its better for it to go on left hand side and fixed values on the right unless the developer intends to override the values. In this case, I assumed we never wanted to override the 'end' value with the spreading of this.pot.